config/sky-pages/sky-pages.config.js

```diff
@@ -71,7 +71,7 @@ module.exports = {
         },
         command: command,
         componentsPattern: '**/*.component.ts',
-        componentsIgnorePattern: './public/**/*',
+        componentsIgnorePattern: '',
```

Comment:
We could make this property optional (in `@skyux/config`) and remove this entirely, if desired.